pkg/kubelet/dockertools/manager.go

```diff
@@ -578,8 +578,8 @@ func (dm *DockerManager) runContainer(
 			labels[kubernetesContainerLabel] = container.Name
 		}
 	}
-	memoryLimit := container.Resources.Limits.Memory().Value()
-	cpuShares := milliCPUToShares(container.Resources.Limits.Cpu().MilliValue())
+	memoryLimit := container.Resources.Requests.Memory().Value()
+	cpuShares := milliCPUToShares(container.Resources.Requests.Cpu().MilliValue())
```

Comment:
Oh I realized that my code is incorrect. memoryLimit is supposed to use Limits, but shares is supposed to use Requests.

What kind of test did you have in mind?